lib/microservices_router.rb

```diff
@@ -0,0 +1,12 @@
+require 'net/http'
+require 'uri'
+
+module MicroservicesRouter
```

Comment:
Slightly odd to have a module with the same name as the app.

Perhaps `ServiceChangeNotifier` or something like that?